Client/Frontend/Browser/ZoomPageBar.swift

```diff
@@ -102,6 +108,8 @@ class ZoomPageBar: UIView {
 
     override func layoutSubviews() {
         super.layoutSubviews()
+        setupGradient()
```

Comment:
Re-making the gradient with each layout pass sounds expensive. Can we just do it if the device rotates or the screen changes size (multitasking on iPad)?